analyzers/src/SonarAnalyzer.Vsix/source.extension.vsixmanifest

```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="SonarAnalyzer.99d84a73-1512-4999-adcc-29d37452719a" Version="8.20.0.0" Language="en-US" Publisher="SonarSource SA" />
+        <Identity Id="SonarAnalyzer.99d84a73-1512-4999-adcc-29d37452719a" Version="8.21.0.0" Language="en-US" Publisher="SonarSource SA" />
         <DisplayName>SonarAnalyzer for CSharp and VB.NET</DisplayName>
         <Description xml:space="preserve">This extension provides on-the-fly feedback to developers on new bugs and quality issues injected into C# and VB.Net code.</Description>
         <MoreInfo>http://vs.sonarlint.org</MoreInfo>
-        <ReleaseNotes>https://github.com/SonarSource/sonar-dotnet/releases/tag/8.20.0.0</ReleaseNotes>
+        <ReleaseNotes>https://github.com/SonarSource/sonar-dotnet/releases/tag/8.21.0.0</ReleaseNotes>
```

Comment:
I realize now that this link is broken - I'm not sure we care though for this VSIX project
But maybe we should remove this ?

Reply:
This reminds me similar question here: https://github.com/SonarSource/sonar-dotnet/pull/3608#discussion_r487819710

Reply:
As far as I can tell this is not used at all since we don't deploy the Vsix project. Do you remember if it was used in the past?

Reply:
Let's merge it as is, no harm is done.
Let's spawn a tooling issue to remove it in the future (and see if something breaks without blocking development).

Reply:
Actually, since this is not used, I've removed the release notes link and updated the script which changes the version.

Reply:
I think it's fine to remove it

Reply:
I don't know what/if it was used for; I would imagine this VSIX was cloned from SLVS and this bit wasn't used at all